doc/nrf/libraries/caf/ble_bond.rst

```diff
@@ -0,0 +1,62 @@
+.. _caf_ble_bond:
+
+CAF: Bluetooth LE bond module
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |ble_bond| of the :ref:`lib_caf` (CAF) allows to introduce Bluetooth LE bonding to the application.
+The module also allows to delete bonds for the default Bluetooth local identity.
+
+Configuration
+*************
+
+Before using the module make sure that the following Kconfig options are enabled:
+
+* :kconfig:`CONFIG_BT_BONDABLE`
+* :kconfig:`CONFIG_BT_SETTINGS`
+* :kconfig:`CONFIG_CAF_SETTINGS_LOADER`
+* :kconfig:`CONFIG_CAF_BLE_COMMON_EVENTS`
+
+The device must be Bluetooth bondable and bonds must be stored in non-volatile memory (settings).
```

Comment:
```suggestion
The device must require Bluetooth bonding and bonds must be stored in the non-volatile memory (settings).
```

Reply:
Device must be bondable. It does not have to require bonding actually.

Reply:
Added missing `the`